UndertaleModLib/UndertaleModLib.csproj

```diff
@@ -30,6 +30,7 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="PresentationCore" />
```

Comment:
PresentationCore is a part of WPF as well and should be removed from here too - I don't think it's even used anywhere

Reply:
Done.